app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from typing import List
```

Comment:
Not a mistake, just reminding that as of python 3.9+ you may use build-in `list` object for type annotations.